docs/go/introduction.md

```diff
@@ -6,14 +6,31 @@ sidebar_label: Go SDK introduction
 
 ## Quick start
 
-1. Add the [Temporal Go SDK](https://github.com/temporalio/sdk-go) to your project:
+1. Clone the Hello World project which has all the basic files explained in our [Hello World Tutorial](/docs/go/hello-world-tutorial):
 
-```
-go get go.temporal.io/sdk@latest
+```bash
+git clone https://github.com/temporalio/hello-world-project-template-go
 ```
 
 2. [Install and run the Temporal Server](/docs/server/quick-install) using `docker compose`.
 
+```bash
+git clone https://github.com/temporalio/docker-compose.git
```

Comment:
Can we try to use the quick-install component being added here: https://github.com/temporalio/documentation/pull/548 ?

Reply:
i had a look but its a little too wordy for the style of this tutorial tbh